modules/library/http/src/test/java/org/geotools/http/HTTPClientFinderTest.java

```diff
@@ -118,8 +117,8 @@ public void createClientWithSystemProxyHost() throws Exception {
             System.setProperty("http.proxyHost", "http://proxy.dummy/");
         }
         try {
-            HTTPClientFinder.createClient(HTTPProxy.class);
-            Assert.fail("Should throw HTTPFactoryException exception.");
+            HTTPClient client = HTTPClientFinder.createClient(HTTPProxy.class);
+            assertTrue(client instanceof SimpleHttpClient);
```

Comment:
Thanks. Is the try/catch still necessary? The test would pass if a ``HTTPFactoryException`` is thrown, I don't think it's what we want, right?